CHANGELOG.next.md

```diff
@@ -65,6 +65,7 @@ Thanks, you're awesome :-) -->
 * Added support for marking fields, field sets, or field reuse as beta in the documentation. #1051
 * Added support for `constant_keyword`'s optional parameter `value`. #1112
 * Added component templates for ECS field sets. #1156, #1186, #1191
+* Added functionality for merging custom and core multi-fields. #982
```

Comment:
Can you move the entry to the `Added` section under `Unreleased` above?

We're trialing a new process for CHANGELOG.next: one section for unreleased and a second for the current feature freeze. Once the 1.8 changes are added to the CHANGELOG as part of the upcoming release, we'll remove them from CHANGELOG.next

Reply:
I think we could backport this to 1.8, no? It doesn't change anything in the schema.

Reply:
True, sounds good to me. Let's take that direction.